SmartSyncExplorerSwift/SmartSyncExplorerSwift.xcodeproj/project.pbxproj

```diff
@@ -0,0 +1,560 @@
+			);
+			path = ViewControllers;
+			sourceTree = "<group>";
+		};
+		B76D656820B3323A00B55764 /* Helpers */ = {
+			isa = PBXGroup;
+			children = (
+				B76D656920B3323A00B55764 /* Constants.swift */,
+				B76D656A20B3323A00B55764 /* UIFont+helper.swift */,
+				B76D656B20B3323A00B55764 /* UIColors+helper.swift */,
+				B76D656C20B3323A00B55764 /* ContactHelpers.swift */,
+			);
+			path = Helpers;
+			sourceTree = "<group>";
+		};
+		B76D657120B332D400B55764 /* Views */ = {
+			isa = PBXGroup;
+			children = (
+				B76D657220B332D400B55764 /* ContactTableViewCell.swift */,
+			);
+			path = Views;
+			sourceTree = "<group>";
+		};
+		B76D65DF20B4AECF00B55764 /* Resources */ = {
+			isa = PBXGroup;
+			children = (
+				B767370020AB57C600F04103 /* userstore.json */,
+				B767370120AB57C600F04103 /* usersyncs.json */,
+				B76D655B20B3314A00B55764 /* Assets.xcassets */,
+				B76D65D120B384B000B55764 /* SalesforceSans-Bold.ttf */,
+				B76D65CF20B384B000B55764 /* SalesforceSans-BoldItalic.ttf */,
+				B76D65CC20B384B000B55764 /* SalesforceSans-Italic.ttf */,
+				B76D65CD20B384B000B55764 /* SalesforceSans-Light.ttf */,
+				B76D65CE20B384B000B55764 /* SalesforceSans-LightItalic.ttf */,
+				B76D65D020B384B000B55764 /* SalesforceSans-Regular.ttf */,
+				B76D65D220B384B100B55764 /* SalesforceSans-Thin.ttf */,
+				B76D65D320B384B100B55764 /* SalesforceSans-ThinItalic.ttf */,
+			);
+			path = Resources;
+			sourceTree = "<group>";
+		};
+		DDFDDF0A29E4BDD114B51BBC /* Pods */ = {
+			isa = PBXGroup;
+			children = (
+				762732C14CFE40A989C3AD70 /* Pods-SmartSyncExplorerSwift.debug.xcconfig */,
+				045D1C5BC2F1EADFDFB3671E /* Pods-SmartSyncExplorerSwift.release.xcconfig */,
+			);
+			name = Pods;
+			sourceTree = "<group>";
+		};
+/* End PBXGroup section */
+
+/* Begin PBXNativeTarget section */
+		4FD6C4731B754AF1002F9F90 /* SmartSyncExplorerSwift */ = {
+			isa = PBXNativeTarget;
+			buildConfigurationList = 4FD6C4931B754AF1002F9F90 /* Build configuration list for PBXNativeTarget "SmartSyncExplorerSwift" */;
+			buildPhases = (
+				FF74F0F7AF3E58C14EBE5178 /* [CP] Check Pods Manifest.lock */,
+				4FD6C4701B754AF1002F9F90 /* Sources */,
+				4FD6C4711B754AF1002F9F90 /* Frameworks */,
+				4FD6C4721B754AF1002F9F90 /* Resources */,
+				68C67C5E03B67EF3A0901D3D /* [CP] Copy Pods Resources */,
+				7056FC8D1D194E5113988767 /* [CP] Embed Pods Frameworks */,
+			);
+			buildRules = (
+			);
+			dependencies = (
+			);
+			name = SmartSyncExplorerSwift;
+			productName = SmartSyncExplorerSwift;
+			productReference = 4FD6C4741B754AF1002F9F90 /* SmartSyncExplorerSwift.app */;
+			productType = "com.apple.product-type.application";
+		};
+/* End PBXNativeTarget section */
+
+/* Begin PBXProject section */
+		4FD6C46C1B754AF1002F9F90 /* Project object */ = {
+			isa = PBXProject;
+			attributes = {
+				LastSwiftUpdateCheck = 0710;
+				LastUpgradeCheck = 0920;
+				ORGANIZATIONNAME = Salesforce;
```

Comment:
That should match templateOrganizationName in template.js: SmartSyncExplorerSwiftOrganizationName